src/_components/About/Values.jsx

```diff
@@ -0,0 +1,16 @@
+export default function Values({ comp, values }) {
```

Comment:
Can you change the text-align to left on mobile for these as well? And maybe have the icons shrink so that they are next to the text like so: 

![image](https://github.com/user-attachments/assets/d5f59c29-006d-45b4-aa45-339eeb864e3f)